source/containers.tex

```diff
@@ -9729,7 +9729,7 @@
 \item It has an \tcode{InputIterator} template parameter and a type that does not qualify as an input iterator is deduced for that parameter.
 \item It has a \tcode{Compare} template parameter and a type that qualifies as an allocator is deduced for that parameter.
 \item It has a \tcode{Container} template parameter and a type that qualifies as an allocator is deduced for that parameter.
-\item It has an \tcode{Allocator} template parameter and a type that does not qualify as an allocator is deduced for that parameter.
```

Comment:
This deletion is at odds with the paper, and the paper is at odds with the status quo. @Quuxplusone, could you perhaps check if the paper is misrepresenting the status quo and was missing this item? (And the item is meant to be retained?)

Reply:
Never mind, I misread the diff. All good!